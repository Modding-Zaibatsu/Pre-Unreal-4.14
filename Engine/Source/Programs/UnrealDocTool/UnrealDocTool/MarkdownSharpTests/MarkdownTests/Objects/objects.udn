[OBJECT:TopicList]
	[PARAM:icon]
		![](%ROOT%/start_icon.png)(convert:false)
	[/PARAM]
	[PARAM:title]
		Test title
	[/PARAM]
	[PARAM:description]
		Test description.

		[OBJECT:TopicList]
			[PARAM:icon]
				![](%ROOT%/start_icon.png)(convert:false)
			[/PARAM]
			[PARAM:title]
				Test title
			[/PARAM]
			[PARAM:description]
				Test description.
			[/PARAM]
			[PARAM:links]
				* [Test link A](Support/Builds) single list item in param
			[/PARAM]
		[/OBJECT]
	[/PARAM]
	[PARAM:links]
		* [Test link A](Support/Builds)
		* [Test link B](GettingStarted/Basics/RecommendedSpecifications)
		* [Test link C](GettingStarted/Basics/DownloadingUnrealEngine)
	[/PARAM]
[/OBJECT]