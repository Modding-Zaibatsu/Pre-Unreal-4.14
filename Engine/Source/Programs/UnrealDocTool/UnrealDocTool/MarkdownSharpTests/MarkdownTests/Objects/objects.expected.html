<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        <div class="topic">
            <div class="icon">
                <p><img alt="start_icon.png" src="./../../images\.\start_icon.Png" /></p>
            </div>

            <div class="wrapper transparent">
                <div class="largetitle">
                    <p>Test title</p>
                </div>
                
                <p>Test description.</p>
                
                <div class="topic">
                    <div class="icon">
                        <p><img alt="start_icon.png" src="./../../images\.\start_icon.Png" /></p>
                    </div>
                    
                    <div class="wrapper transparent">
                        <div class="largetitle">
                            <p>Test title</p>
                        </div>
                        
                        <p>Test description.</p>
                        
                        <div class="arrowlist">
                            <ul>
                                <li>
                                    <p><a href="..\Support\Builds\index.html">Test link A</a> single list item in param</p>
                                </li>
                            </ul>
                        </div>
                    </div>
                </div>
                
                <div class="arrowlist">
                    <ul>
                        <li>
                            <p><a href="..\Support\Builds\index.html">Test link A</a></p>
                        </li>
                        <li>
                            <p><a href="..\GettingStarted\Basics\RecommendedSpecifications\index.html">Test link B</a></p>
                        </li>
                        <li>
                            <p><a href="..\GettingStarted\Basics\DownloadingUnrealEngine\index.html">Test link C</a></p>
                        </li>
                    </ul>
                </div>
            </div>
        </div>
    </body>
</html>
